src/manifests.rs

```diff
@@ -51,12 +51,13 @@ pub fn package_name(current_dir: &Path) -> Result<String, Error> {
 /// Returns absolute path for a current directory.
 pub fn get_absolute_path(current_dir: &str) -> Result<String, Error> {
```

Comment:
Should we name the argument as "relative_path" or something other than "current_dir" to make it more generic?

so the comment can change to "Returns absolute path of a directory."